CIPs/0021.md

```diff
@@ -0,0 +1,64 @@
+---
+cip: <to be assigned>
+title: Governable lookback window
+author: Andres di Govanni (@andres-dg)
+discussions-to: <to be assigned>
```

Comment:
please create a github issue for your proposal called "Discussions for Governable Lookback Window"

Reply:
@andres-dg just made it for you https://github.com/celo-org/celo-proposals/issues/86